### PR TITLE
40 tour ticket card

### LINKED ISSUE #40: Tour - implement tour ticket cards block
Mobile view:
Screenshot 2023-05-05 at 14 25 47

Desktop view:
Screenshot 2023-05-05 at 14 26 05

The tour ticket card can contain:
1. Image (optional)
2. Title (optional)
3. Subtitle (optional)
4. Price text (optional)
5. CTA Link 
6. additional info text (optional)
7. More info link

Some variations:
Screenshot 2023-05-05 at 14 28 02
Screenshot 2023-05-05 at 14 27 34
Screenshot 2023-05-05 at 14 26 48

### PR DESCRIPTION
**Pull request by msagolj**
- Created a block ticket-card-list , that generates the complete list of ticket cards for all the variations (main category in hero, sub category in hero, main in related tickets block, sub in related tickets block)
- Extended hero tours to support sub category listing (combo) if configured
- Created a block related tours that lists tours by category/sub category.

MISSING:
- the swiping of the sub categories. will be added in a separate issue.

Fix #40

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/drafts/mirko/tour-bernabeu
- After: https://40-tour-ticket-card--realmadrid--hlxsites.hlx.live/drafts/mirko/tour-bernabeu

### CODE REVIEW

**Review by chicharr — APPROVED** [review]
Added some comments to be addressed in a separate ticket

File: sites/blocks/hero-tour/hero-tour.css [hunk 1]
```diff
@@ -166,8 +170,35 @@ body.tour .section.hero-tour-container .hero-tour.block .content .promo strong {
...
+
+/* sub category title */
+.hero-tour.block .content .sub-wrapper .sub-cat-title {
+  margin: 0 1.25rem 1.5625rem;
+  font-family: var(--heading-font-family);
+  font-size: 1.6875rem;
+  font-weight: 400;
+  line-height: 1.11;
+  letter-spacing: 0.07rem;
+  color: #fff;
```
[the remark this answers is not in the document]
Comment (msagolj) [reply sites/blocks/hero-tour/hero-tour.css:186]: fixed as part of #136

File: sites/blocks/hero-tour/hero-tour.js [hunk 2]
```diff
@@ -104,28 +106,37 @@ export default async function decorate(block) {
...
+    await loadBlock(ticketCardListBlock.firstElementChild);
+    contentWrapper.append(ticketCardListBlock);
+  }
+
+  // if sub categories are defined
+  if (tourSubCategory) {
+    const subCatContainer = document.createRange().createContextualFragment(`
+    <div class='sub-wrapper'>
+      <h2 class='sub-cat-title'>
+        <span class='sub-cat-subtitle'>También podéis elegir</span>
```
[the remark this answers is not in the document]
Comment (msagolj) [reply sites/blocks/hero-tour/hero-tour.js:128]: fixed as part of #136

File: sites/blocks/hero-tour/hero-tour.css [hunk 3]
```diff
@@ -233,54 +264,714 @@ body.tour .section.hero-tour-container .hero-tour.block .background picture.desk
...
+  letter-spacing: 0.01rem;
+  color: #444;
+}
+
+/* the info link */
+.hero-tour.block .ticket-card-list.hero.sub .info-link {
+  font-size: 0.8125rem;
+  line-height: 1;
+  color: #0067d1;
+}
```
[the remark this answers is not in the document]
Comment (msagolj) [reply sites/blocks/hero-tour/hero-tour.css:977]: as discussed its needed due to the many breakpoints

**Pull request merged**